src/main/java/edu/indiana/dlib/amppd/model/Collection.java

```diff
@@ -25,7 +27,7 @@
 @ToString(callSuper=true, onlyExplicitlyIncluded=true)
 public class Collection extends Content {
 
-	@OneToMany(mappedBy="collection")
+	@OneToMany(mappedBy="collection", fetch = FetchType.EAGER)
     private Set<Item> items; 
```

Comment:
I wonder if making fetch EAGER could impact performance, if there're many items in a collection (in production it would be ). I remember having to add something extra to fetch the reference if not EAGER.

Reply:
I was able to solve  the exception "org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: edu.indiana.dlib.amppd.model.Collection.items, could not initialize proxy - no Session" I was getting when using lazy loading.  The key was to mark the service class (or whatever is using the repository) as transactional using the @Transactional annotation.  In my case, that meant adding it to the BatchServiceImpl and BatchValidationServiceImpl classes.  We will see if there are any additional side affects to that, but it seems to work for now.

Reply:
Yep, I remember that was exactly the error I got and the solution I found. thanks!